flow-libs/browserslist.js.flow

```diff
@@ -0,0 +1,13 @@
+// @flow strict-local
+
+declare module 'browserslist' {
+  import type {ConfigResult} from '@parcel/types';
+
+  declare type BrowserslistConfig = {|[string]: string | Array<string>|};
+
+  declare module.exports: {|
+    (browsers: string | Array<string>): Array<string>,
+    parseConfig(configResult: ConfigResult): Promise<?BrowserslistConfig>,
```

Comment:
looks like this takes a string: https://github.com/browserslist/browserslist/blob/88f7d5141f9be0ac3d712f8b52f4654fadbdeeff/node.js#L260